kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/orchestration/ContainerOrchestrationService.java

```diff
@@ -41,7 +41,37 @@ public interface ContainerOrchestrationService {
      * @return a list of {@link ContainerInstanceDescriptor}s representing the available containers
      */
     public List<ContainerInstanceDescriptor> listContainerDescriptors();
-
+    
+    /**
+     * @since 2.4
+     * Lists all available images, returning the corresponding {@link ImageInstanceDescriptor}s
+     *
+     * @return a list of {@link ContainerInstanceDescriptor}s representing the available containers
+     */
+    public List<ImageInstanceDescriptor> listImageInstanceDescriptor();
```

Comment:
I think the javadoc should look like this:
/**
     * Lists all available images, returning the corresponding {@link ImageInstanceDescriptor}s
     *
     * @return a list of {@link ContainerInstanceDescriptor}s representing the available containers
     * @ since 2.4
     */

Reply:
Furthermore, I think we should align to the convention we have that if we return a list, the method should be listImageInstanceDescriptors